src/components/account/ApproveUser.vue

```diff
@@ -52,7 +58,24 @@ import {accountService} from '@/service/account-service';
 			errors: [],
 			id: null,
 			approve_user: false,
-      reject_user: false
+      reject_user: false,
+      approve_user_emailId:null,
+      approve_user_status:null
+    }
+  },
+computed:{
+    isRequested: function(){
+      if(this.approve_user_status=='REQUESTED')
+      {
+          return true;
+      }
+      return false;
+    },
+    isUserApproved: function(){
+      if(this.approve_user_status == 'ACTIVATED' || this.approve_user_status == 'ACCEPTED'){
+          return true;
+      }
+      return false;
     }
   },
```

Comment:
similarly, return this.approve_user_status == 'ACTIVATED' || this.approve_user_status == 'ACCEPTED'